include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -2074,12 +2073,26 @@ FEPointEvaluationBase<n_components_, dim, spacedim, Number>::do_reinit()
           }
         else
           {
-            shapes.resize_fast(n_q_batches * n_shapes);
```

Comment:
This line (and a similar one for the faces above) is only tangentially related to the combination of points. However, it is still crucial: Even though `resize_fast` does not do much, https://github.com/dealii/dealii/blob/81ec8648500a910743e28cef437c843d84a469e6/include/deal.II/base/aligned_vector.h#L1298-L1304 when the size is already correct, we should nonetheless only do the operation once outside the loop over `n_q_batches`, not in every loop iteration.